docs/root/intro/version_history.rst

```diff
@@ -1,6 +1,24 @@
 Version history
 ---------------
 
+0.3.0 (Mar 26,2020)
+===================
+
+This is the first release of Envoy Mobile we are tagging as production ready 🎉
+
+Since early November, when the team tagged v0.2.0, we have been hard at work to stabilize the library,
+and harden it via experiments with Lyft's Alpha and Beta releases. We have released our production
+binaries with Envoy Mobile for a couple weeks now, and are starting to expose a percentage of our
+production clients to Envoy Mobile with this release.
+
+Since v0.2.3 we have largely focused on observability (#701, #702, #718, #722, #751, #768, #771)
+
+Additional changes:
+
+- Fixes iOS on trailers missing (#703)
```

Comment:
```suggestion
- Fixes trailers missing on iOS (#703)
```

Reply:
I was referring to the callback, but I think this works better